markdown-to-confluence/main.js

```diff
@@ -0,0 +1,253 @@
+import { mkdir, readFile, writeFile } from 'node:fs/promises';
+import os from 'node:os';
+import path from 'node:path';
+import axios from 'axios';
+import { ConfluenceClient } from 'confluence.js';
```

Comment:
That's confusing :)
Was wondering why I don't see a confluence.js file, but I see it's a package https://www.npmjs.com/package/confluence.js